.pre-commit-config.yaml

```diff
@@ -12,6 +12,7 @@ repos:
       - id: check-json
       - id: check-yaml
       - id: pretty-format-json
+        files: \.(json|json5|js)$
```

Comment:
Maybe instead of specifying what to include, exclude the `.ipynb` file format since we don't want that to be prettified anyways.